app/scripts/controllers/bridge/bridge-controller.ts

```diff
@@ -44,9 +44,23 @@ export default class BridgeController extends BaseController<
 
   setBridgeFeatureFlags = async () => {
     const { bridgeState } = this.state;
-    const bridgeFeatureFlags = await fetchBridgeFeatureFlags();
-    this.update((_state) => {
-      _state.bridgeState = { ...bridgeState, bridgeFeatureFlags };
-    });
+    try {
+      const bridgeFeatureFlags = await fetchBridgeFeatureFlags();
+      this.update((_state) => {
+        _state.bridgeState = { ...bridgeState, bridgeFeatureFlags };
+      });
+    } catch (error) {
+      console.error(
+        'Failed to fetch Bridge feature flags, using defaults.',
+        error,
+      );
+      this.update((_state) => {
```

Comment:
Maybe it would be better to leave flags unchanged in this situation? Unless you think there's a risk in a user getting "stuck" with certain flags enabled that you wanted to disable remotely.

Reverting to defaults every failure might lead to this flipping back and forth often, if the user has a spotty network connection.

Reply:
> Unless you think there's a risk in a user getting "stuck" with certain flags enabled that you wanted to disable remotely

I did this to prevent users from using the native Bridge experience if it likely won't work as expected and we need to disable it (for example: during outages, or if user has moved from an unblocked to a blocked region).

However, I realize that this is probably not needed because subsequent bridge-api calls would fail even if the user manages to load the experience. One disadvantage to not reverting to default values is that the user won't be redirected to Portfolio (fallback behavior) if the Bridge experience only needs to be disabled in extension. 

I added a task for our team to test this scenario before launch to see if the experience is acceptable. But I think for now I should remove the reset to avoid flipping back and forth